cpp/src/utilities/error_utils.h

```diff
@@ -152,6 +152,7 @@ inline void check_stream(cudaStream_t stream, const char* file,
  * GDF_REQUIRE should be considered deprecated.
  *
  *---------------------------------------------------------------------------**/
+#ifndef CUDA_TRY
```

Comment:
Can't we just include

https://github.com/rapidsai/cudf/blob/branch-0.13/cpp/include/cudf/utilities/error.hpp

?

Reply:
@afender ?

I think the goal was to minimize dependencies on cudf, although I'm not sure that's necessary in this case.  I had to add the #ifndef in a couple of places in case both were included.  My code cleanup work today might eliminate that issue.